### PR TITLE
update documentation for --static-bindings option

### PR DESCRIPTION
**Pull request by slavchev**
Make it explicit that `--static-bindings` is an experimental feature.

### CODE REVIEW

**Comment by slavchev** [comment]
ping @ikoevska

**Comment by Fatme** [comment]
:+1:

File: docs/man_pages/project/testing/build-android.md [hunk 1]
```diff
@@ -14,7 +14,10 @@ Builds the project for Android and produces an APK that you can manually deploy
 * `--key-store-password` - Provides the password for the keystore file specified with `--key-store-path`. You can use the `--key-store-*` options along with `--release` to produce a signed release build. You need to specify all `--key-store-*` options.
 * `--key-store-alias` - Provides the alias for the keystore file specified with `--key-store-path`. You can use the `--key-store-*` options along with `--release` to produce a signed release build. You need to specify all `--key-store-*` options.
 * `--key-store-alias-password` - Provides the password for the alias specified with `--key-store-alias-password`. You can use the `--key-store-*` options along with `--release` to produce a signed release build. You need to specify all `--key-store-*` options.
-* `--static-bindings` - If set, generates static bindings from your JavaScript code to corresponding native Android APIs during build. This static bindings speed up app loading.
+* `--static-bindings` - This is an experimental feature. If set, generates static bindings from your JavaScript code to corresponding native Android APIs during build. This static bindings speed up app loading.[\*\*](#note)
```
Comment (ikoevska) [inline docs/man_pages/project/testing/build-android.md]: Bold the "This is an experimental feature part."

-> If set, generates static bindings from your JavaScript code to the corresponding native Android APIs during build...

File: docs/man_pages/project/testing/build-android.md [hunk 2]
```diff
@@ -14,7 +14,10 @@ Builds the project for Android and produces an APK that you can manually deploy
...
+
+<% if(isHtml) { %><a id="note"></a><% } %>
+\*\* By default NativeScript runtime for Android uses runtime binding generator. When you extend a Java class and overwrite a lot of methods this could be a potentially slow operation.
```
Comment (ikoevska) [inline docs/man_pages/project/testing/build-android.md]: By default, the NativeScript runtime for Android.

-> comma in "a lot of methods, this could be"

**Comment by ikoevska** [comment]
:+1: after addressing my comments.

**Pull request merged**